app/models/page.rb

```diff
@@ -1,8 +1,10 @@
 class Page < ApplicationRecord
+  TEMPLATE_OPTIONS = %w[contained full_within_layout full_page].freeze
```

Comment:
Actually, `full_within_layout` and `full_page` work the same for now: https://github.com/thepracticaldev/dev.to/blob/17a5e831fd56849bcdb5d99c736c82888105a741/app/views/pages/show.html.erb#L21-L35
We may want to remove one of these or change the behavior in the future.

Reply:
Yeah. `full_page` is designed to be _the entire page_ and `full_within_layout` is supposed to be full page but still showing the header and footer.

..... Since we now use serviceworkers to display the header and footer it will be more technically difficult to actually show the `full_page` version. I'd be down to remove one of these and only use "full within layout" approach. And since we only have one now, I'd say just sticking to `full_page` to mean that is a good approach.

Does that make sense?

Reply:
The intentions are clear but as they are not used this way currently, it doesn't make much sense to implement the `full_page`, so I agree that we can remove it for now.